src/algorithm/extremes.rs

```diff
@@ -282,7 +279,7 @@ mod test {
     #[test]
     fn test_polygon_extreme_wrapper_convex() {
         // convex, with a bump on the top-right edge
-        let mut points_raw =
+        let points_raw =
```

Comment:
@urschrei  Sorry, what reverse call on line 288? I don't see `reverse` anywhere in this file.